generator/FORMAT.md

```diff
@@ -25,6 +26,10 @@ module_name:
      # as a label value on that gauge.
 
      # A metric that's part of a table, and thus has labels.
```

Comment:
This isn't part of a table. You don't need to change this file.

Reply:
I was thinking it would be worth having an example in the FORMAT file. Don't care either way.

Reply:
The format file is more the general structure, the various types are over in the readme.

Reply:
Got it, removed.

Reply:
The types used to be in here, but they got moved when the type override was added. I'd prefer not to having to maintain two lists of types, as it's more likely they'll get out of sync.